flocker/node/_docker.py

```diff
@@ -29,6 +30,9 @@
     RestartNever, RestartAlways, RestartOnFailure, pset_field, pvector_field)
 
 
+logger = logging.getLogger(__name__)
```

Comment:
Ah, I see.  Hm.  Ideally this stuff would continue to show up somewhere.  My motivation is that I want some evidence that when we hit this case the work-around being applied here is actually addressing the problem.  Messages in a log file somewhere showing us that we hit it would make good evidence.

As far as stderr being the only place where the log output is visible on buildbot... `test.log` is also available, and I think in general anything that we manage to get into `test.log` will also end up in the application-specific log file under real usage (`/var/log/flocker/whatever.log` for example).

So, I agree that stderr is not a great choice for the "where".  I think this means the ideal is indeed to switch over to Eliot and I think that's probably not so much work that we'll benefit from skipping that work now.  If that seems reasonable, please go ahead with that and then merge.